### PR TITLE
Remove failery jobs

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- remove failery jobs

**Related issue(s)**

https://github.com/kyma-project/kyma/pull/13130

### CODE REVIEW

**Comment by Halamix2** [comment]
/hold
until main PR is merged

**Review by adamwalach — APPROVED** [review]

**Pull request merged**
